features/text-box.yml.dist

```diff
@@ -3,10 +3,12 @@
 
 status:
   baseline: false
-  support:
-    safari: "18.2"
-    safari_ios: "18.2"
+  support: {}
 compat_features:
+  # baseline: false
+  # support:
+  #   safari: "18.2"
+  #   safari_ios: "18.2"
```

Comment:
https://developer.chrome.com/blog/css-text-box-trim says that chrome/edge 133 have support for text-box too. And BCD seems to agree. Do you know why Safari is the only supported browser here?

Reply:
There are conflicts in this file, https://github.com/web-platform-dx/web-features/blob/main/features/text-box.yml.dist looks good on main and I bet that resolving the conflicts will fix this.